### DISCUSSION TITLE
PHP8 is fine 👍

### DISCUSSION THREAD

**Discussion by anders8** [getgrav/grav#3733] [General]
I recently updated some backend server stuff, and went from PHP 7x to 8x. Prior to doing so I searched here for any issues, found only a note about incompatibility with an old short-code module. Can confirm that the _current_ short-code module, as of July 2023, doesn't have whatever issue that was back then.

This was nginx, and so I did have to go back and explicitly install some PHP extensions for v8, set up fpm, and so on, but that falls under "getting PHP to work" and there's nothing grav-specific there.

Naturally I went through my sites and updated core, all plugins and themes to the latest version prior to switching PHP.

**Comment by Jogai** [comment]
Well, I'm running a grav site on 8.2 and it seems to have some sort of memory problem. Maybe running on 8 is my problem

**Reply by anders8** [reply]
Can't speak to 8.2, the OS upgrade brought in 8.0.29 as its default. I'd be surprised if Grav was doing anything super wacky that wasn't effecting a zillion other site's tech though. Good luck buddy, let us know if you find anything!

**Reply by Jogai** [reply]
switching back to 7 didnt help so far, so that's not it...

**Comment by rhukster** [comment]
Tested and approved to work with PHP 8.2. What is the error you are getting?
